src/main/native/com/code_intelligence/jazzer/android/BUILD.bazel

```diff
@@ -0,0 +1,71 @@
+load("//bazel:compat.bzl", "SKIP_ON_WINDOWS")
+
+genrule(
+    name = "android_jvmti_h_gen",
+    outs = ["jvmti.h"],
+    cmd = "curl --silent https://android.googlesource.com/platform/art/+/1cff8449bac0fdab6e84dc9255c3cccd504c1705/openjdkjvmti/include/jvmti.h?format=TEXT | base64 --decode > $(OUTS)",
```

Comment:
Build actions must not perform network I/O. You can download this file using `http_file` in `repositories.bzl`, see https://github.com/fmeum/rules_jni/blob/749bda09e678c9465c3f4ed62d26b925ca89e96e/jni/internal/repositories.bzl#L39-L45 for an example.

Reply:
I only know to get this file from here:

https://android.googlesource.com/platform/art/+/1cff8449bac0fdab6e84dc9255c3cccd504c1705/openjdkjvmti/include/jvmti.h?format=TEXT

and it comes base64 encoded. Do you know an easy way to reference this Bazel? That's why I was using `curl --silent <url> | base64 --decode > $(OUTS)`

Reply:
You could do this in a two-step process: First, add an `http_file` to `repositories.bzl` that gets the Base64-encoded file. Then, in a `genrule`, decode it. That keeps the build action hermetic.